### PR TITLE
Scala 3 uses * as wildcard in imports

### PR DESCRIPTION
**Pull request by xhudik**
replaced `_` with `*` in imports

### CODE REVIEW

**Review by b-studios — COMMENTED** [review]
Thanks for fixing it, I was going to do that today :)

Could you maybe add a link to the reference? 
https://dotty.epfl.ch/docs/reference/changed-features/imports.html

Also have you checked the rest of book and documents for imports?

I `grep`ped for it and found old imports in the following files:

- `_overviews/scala3-macros/tutorial/quotes.md`
- `_overviews/scala3-macros/tutorial/reflection.md`
- `_overviews/scala3-book/domain-modeling-fp.md`
- `_overviews/scala3-book/domain-modeling-oop.md`
- `_overviews/scala3-book/interacting-with-java.md`
- `_overviews/scala3-book/methods-main-methods.md`
- `_overviews/scala3-book/packaging-imports.md`
- `_overviews/scala3-book/scala-features.md`
- `_overviews/scala3-book/scala-for-javascript-devs.md`
- `_overviews/scala3-book/taste-modeling.md`
- `_overviews/scala3-book/taste-objects.md`
- `_overviews/scala3-book/taste-toplevel-definitions.md`
- `_overviews/scala3-book/types-opaque-types.md`

**Comment by b-studios** [comment]
If you could apply the same changes also to the above files, I would be very happy. Otherwise, I can go ahead and do it.

**Comment by b-studios** [comment]
Oh, I just noticed that you already have a PR for this (#1944). Thanks!

**Comment by xhudik** [comment]
Hi @b-studios (Jonathan :))  -  yes, i did some changes as I read the book. And yes, I can take your list fix those that I had missed.

**Comment by b-studios** [comment]
Great, thanks! Can you add them to this PR so we have one big PR with all the related changes?

**Comment by xhudik** [comment]
@b-studios - pls check it now

File: _overviews/scala3-book/packaging-imports.md [hunk 1]
```diff
@@ -256,7 +254,7 @@ In the rare event there’s a naming conflict and you need to import something f
 ```
 package accounts
 
-import _root_.users._
+import _root_.accounts.*
```
Comment (b-studios) [inline _overviews/scala3-book/packaging-imports.md:257]: good catch!

**Review by b-studios — APPROVED** [review]

**Pull request merged**